pkg/reconciler/labeler/labels.go

```diff
@@ -90,38 +89,22 @@ func (c *Reconciler) clearLabels(ctx context.Context, ns, name string) error {
 // setLabelForListed uses the accessor to attach the label for this route to every element
 // listed within "names" in the same namespace.
 func setLabelForListed(ctx context.Context, route *v1alpha1.Route, acc accessor, names sets.String) error {
-	nameToAccessor := make(map[string]kmeta.Accessor)
-
-	// Lookup names that are missing our Route label.
 	for name := range names {
```

Comment:
perhaps you want to keep `names.List()` since it's sorted. Not sure if that matters, but it was there for _some_ reason.